dev-tools/jenkins_setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+#!/usr/bin/env bash
+set -euox pipefail
+
+: "${HOME:?Need to set HOME to a non-empty value.}"
```

Comment:
I think this could go into [common.bash](https://github.com/elastic/beats/blob/ce9e47b83ab5a7dd09c99832cdcf0ebdbc485cb0/dev-tools/common.bash) as a function like `jenkins_setup`.